docs/guides/auth/email_password.rst

```diff
@@ -449,6 +449,63 @@ an ``auth_token``.
    };
 
 
+Creating a User object
+----------------------
+
+Some applications may want to create a custom ``User`` type in their default
+module to attach application-specific information. You can tie this to an
+``ext::auth::Identity`` by using the ``auth_token`` in our
+``ext::auth::client_token`` global and inserting your ``User`` object with a
+link to the ``Identity``.
```

Comment:
```suggestion
For some applications, you may want to create a custom ``User`` type in the
default module to attach application-specific information. You can tie this to
an ``ext::auth::Identity`` by using the ``auth_token`` in our
``ext::auth::client_token`` global and inserting your ``User`` object with a
link to the ``Identity``.
```
I know this is a nitpick, but it feels off to me to say that an application "wants" something. Tweaked the language just a bit to instead talk about what the developer wants.